### PR TITLE
release prep for v0.1.12

### PR DESCRIPTION
**Pull request by mgeplf**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
+## v0.1.12:
+
+### Added:
+ - Add a check when feeding a directory path to readFile (#188)
+ - Add support for IO-blocks (#183)
+
+### Fixed:
+ - Bump HighFive to v2.4.0 (#190); fixes errors on archs: aarch64, s390x and ppc64le
+ - Update hdf5 to 1.12.1 in linux wheel building (#191)
+ - With `-Wp,-D_GLIBCXX_ASSERTIONS`, tests fail with an assertion error of
```
Comment (matz-e) [inline CHANGELOG.md:12]: of what?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,21 @@
...
+ - Improve mem. ops. + NodeID-ElementID mapping bindings (#179)
```
Comment (matz-e) [inline CHANGELOG.md:13]:
```suggestion
 - Improve memory operations + NodeID-ElementID mapping bindings (#179)
```

File: CHANGELOG.md (hunk 1 above)
Comment (mgeplf) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 2 above)
Comment (mgeplf) [reply CHANGELOG.md]: added, thanks

**Review by sergiorg-hpc — APPROVED** [review]
Amazing! Thanks @mgeplf, I will create a Spack PR for it as well and add you as reviewer.

**Pull request merged**